process_pr.py

```diff
@@ -1239,8 +1252,12 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
         elif re.match(r"^\s*" + REGEX_IGNORE_COMMIT_COUNT + r"\s*$", first_line):
             comment_emoji = "-1"
             if commenter in CMSSW_ISSUES_TRACKERS:
-                ok_too_many_commits = True
-                comment_emoji = "+1"
+                if pr.commits < TOO_MANY_COMMITS_FAIL_THRESHOLD:
+                    ok_too_many_commits = True
+                    comment_emoji = "+1"
+                else:
+                    warned_too_many_commits = False
```

Comment:
@iarspider , do we need to reset this here? what will happen if this comment is after bot has warned about the too many commits?

Reply:
yes, it's a typo - I wanted to reset `ok_too_many_commits`